helper/krylovHelper.cpp

```diff
@@ -0,0 +1,151 @@
+#include <vector>
+#include <memory>
+#include <iomanip>
+#include <type_traits>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+
+#include "krylovHelper.h"
```

Comment:
Should we maybe also call the file "outputHelper"?

Reply:
We can, but the original idea was to extend the helper class to also handle observable naming (and propagate it to the output file). The helper for the output is only a first (but most helpful) step.

Reply:
Alright :)